services/core/shared/indexdb/mysql.js

```diff
@@ -22,6 +22,7 @@ const logger = Logger();
 
 const connectionPool = {};
 const tablePool = {};
+const defaultColumns = {};
```

Comment:
Please consider renaming `defaultColumns` to `defaultSelectColumns` or something similar that is in synchronous with the change requested above.

> Please consider renaming isDefaultColumn to isDefaultSelect, isResultDefault or includeDefaultResult.

```suggestion
const defaultSelectColumns = {};
```